README.md

```diff
@@ -0,0 +1,61 @@
+# Microformats JavaScript/Node Parser Website
+
+Website for Microformats JavaScript parser (based on pin13.net by [@aaronpk](https://github.com/aaronpk)).
+
+https://node.microformats.io
+
+## Deployment
+
+All commits to the `main` branch get auto-deployed to the live website [running on Heroku](https://node.microformats.io).
+
+## Getting Started
+
+This website is built using [Yarn](https://yarnpkg.com/).
+
+To start working with this project, first clone the repository for this project:
+
+```
+git clone https://github.com/microformats/microformats-parser-website-node.git
+cd microformats-parser-website-node
+```
+
+Next, install the required dependencies and start the server:
+
+```
+yarn install
+yarn run start
+# Or, if you'd like a different port:
+# PORT=5000 yarn run start 
+```
+
+You can view your running local application at this URL:
+
+```
+http://localhost:9080
+```
+
+## Requirements
+
+- [Node 17](https://nodejs.org/en/blog/release/v17.0.0/)
+- [Yarn](https://yarnpkg.com/cli/install)
```

Comment:
Probably should consider yarn optional here as npm does a good enough job installing packages.

Yarn can be a little confusing, particularly given the drastic changes in version 2. I'm okay with recommending it but wouldn't want that to hold someone back from contributing.

Reply:
Given that the project currently has a `yarn.lock` file (and yarn commands), using npm will create another lockfile. I think it's fine to mention here while this package uses Yarn. Dropping yarn should be a seperate discussion.